currency.go

```diff
@@ -203,6 +203,7 @@ var currencies = Currencies{
 	UYU: {Decimal: ".", Thousand: ",", Code: UYU, Fraction: 2, NumericCode: "858", Grapheme: "$U", Template: "$1"},
 	UZS: {Decimal: ".", Thousand: ",", Code: UZS, Fraction: 2, NumericCode: "860", Grapheme: "so\u2019m", Template: "$1"},
 	VEF: {Decimal: ".", Thousand: ",", Code: VEF, Fraction: 2, NumericCode: "928", Grapheme: "Bs", Template: "$1"},
```

Comment:
Note this currency has the wrong numeric code, which would make it fail in case the function `CurrencyByNumericCode` is used, because it has the same value as the newly added VES currency.

https://github.com/Rhymond/go-money/blob/11fb01ee0a561feb7a831160e90b0fdef19b891e/currency.go#L21

The correct value for VEF is 937 according to https://www.six-group.com/dam/download/financial-information/data-center/iso-currrency/lists/list-three.xml (https://www.six-group.com/en/products-services/financial-information/data-standards.html)

Reply:
Thanks! I'm changing that on the PR.

Reply:
Thanks @sk-!